webpack/components/Table/TableWrapper.js

```diff
@@ -86,19 +102,10 @@ const TableWrapper = ({
         <FlexItem>
           {children}
         </FlexItem>
-        <FlexItem align={{ default: 'alignRight' }}>
-          <Pagination
-            itemCount={total}
-            page={page}
-            perPage={perPage}
-            onSetPage={(_evt, updated) => onPaginationUpdate({ page: updated })}
-            onPerPageSelect={(_evt, updated) => onPaginationUpdate({ per_page: updated })}
-            perPageOptions={usePaginationOptions().map(p => ({ title: p.toString(), value: p }))}
-            variant="top"
-          />
-        </FlexItem>
+        {ShowPageControls(PaginationVariant.top)}
```

Comment:
A function that returns JSX is a React component. Should be invoked not with `{...}` but instead
```js
<ShowPageControls variant={PaginationVariant.top} />
```

However, I wouldn't have this as a separate component anyway.

Reply:
Or you may choose to keep it as a separate component.. I don't feel as strongly about that. As long as it's notated correctly ;)